### DISCUSSION TITLE
SimpleStorage Deployment(solidity-section-1)

### DISCUSSION THREAD

**Discussion by suryakshbehera** [Cyfrin/foundry-full-course-cu#1972] [Q&A] [answered]
I'm facing trouble to deploy my simple_storage contract on zkSync. I followed the whole process but instead of simple_storage , my_Token contract is deploying by default( I redeployed also, but same thing is happened).
How do I overcome this?

**Comment by EngrPips** [comment]
Hello @Jeeban369, Please show us your remix so we can see what you are doing.

**Reply by suryakshbehera** [reply]
![Screenshot 2024-07-09 083620](https://github.com/Cyfrin/foundry-full-course-cu/assets/96816170/b5ca842f-8da1-4cd6-94e1-19705ae262eb)
All the above files are automatically generating during compilation.

remix id link-https://remix.ethereum.org/#lang=en&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.22+commit.4fc1097e.js

**Comment by EngrPips** [comment, marked as answer]
Hello @Jeeban369, please switch your `workspace` to `default workspace` and ensure the `SimpleStoarge` contract is inside the `contracts` folder, then retry.

**Reply by suryakshbehera** [reply]
![Screenshot 2024-07-09 112502](https://github.com/Cyfrin/foundry-full-course-cu/assets/96816170/4d134ee8-7181-460e-ba73-ab375e64dc83)

Thank You @EngrPips for your suggestion, I have successfully deployed my contract.

my mistake was->I was using my account address instead of contract address.

**Reply by EngrPips** [reply]
Excellent, Keep crushing it. I am glad you figured it out.
